sqlmesh/core/snapshot/evaluator.py

```diff
@@ -130,6 +130,7 @@ def apply(query_or_df: QueryOrDF, index: int = 0) -> None:
             start=start,
             end=end,
             latest=latest,
+            dbt_is_incremental=snapshot.is_incremental_unmanaged and bool(snapshot.intervals),
```

Comment:
Do you think we need to call it dbt_is_incremental here? We've jumped through a lot of hoops to abstract dbt away from core code. Do you feel its okay to let it into the evaluator?

Reply:
Yeah, at this point I don't feel like this is something we should support outside dbt. This might change in future

Reply:
I agree that this is a dbt compatability mode feature. I don't understand why the kind was named in a generic way, but the param wasn't. More of a nit.

Reply:
i'd rather name it something generic, i could see us supporting this for id based workflows and you don't want to use merge unique